cmd/plugin/rpaasv2/cmd/info.go

```diff
@@ -0,0 +1,89 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"strconv"
+
+	"github.com/olekukonko/tablewriter"
+	"github.com/tsuru/rpaas-operator/pkg/rpaas/client"
+	"github.com/tsuru/rpaas-operator/pkg/rpaas/client/types"
+	"github.com/urfave/cli"
+)
+
+func initInfoFlags() []cli.Flag {
+	return []cli.Flag{
+		cli.StringFlag{
+			Name:     "service, s",
+			Usage:    "service name",
+			Required: true,
+		},
+		cli.StringFlag{
+			Name:     "instance, i",
+			Usage:    "instance name",
+			Required: true,
+		},
+	}
+}
```

Comment:
Similar to the above suggestion, why do you not set these flags directly in the command at Info func?